interface/modules/custom_modules/oe-module-claimrev-connect/public/setup.php

```diff
@@ -0,0 +1,165 @@
+<?php
+    namespace OpenEMR\Modules\ClaimRevConnector;
+    $tab="setup";
+    require_once '../src/ClaimRevModuleSetup.php';
+    use OpenEMR\Modules\ClaimRevConnector\ClaimRevModuleSetup;
+
+    $services = ClaimRevModuleSetup::GetBackgroundServices();
+    if(isset($_POST['deactivateSftp'])) {
+        ClaimRevModuleSetup::DeactivateSftpService();
+    
+    }
+    if(isset($_POST['reactivateSftp'])) { 
+        ClaimRevModuleSetup::ReactivateSftpService();
+    
+    }
+    if(isset($_POST['backgroundService'])) { 
+        ClaimRevModuleSetup::CreateBackGroundServices();
+    
+    }
+
+    $services = ClaimRevModuleSetup::GetBackgroundServices();
+?>
+<html>
+    <head>
+        <link rel="stylesheet" href="../../../../../public/assets/bootstrap/dist/css/bootstrap.min.css">
+    </head>
+<title>ClaimRev Connect - Setup</title>
+<body>
+    <div class="row"> 
+        <div class="col">
+            <?php
+                include '../templates/navbar.php';
+            ?>
+        </div>
+    </div>
+    <div class="row"> 
+        <div class="col">
+            <h1>Setup</h1>        
+        </div>
+    </div>
+    <div class="row"> 
+        <div class="col-6">
+            <div class="card">
+                <ul>
+                    <li>
+                        <h6>x12 Partner Record</h6>
```

Comment:
xlt() all the labels/text in this script. Note that the longer strings needs to be kept as one liners (ugly code, but gotta do it), for example the following:
```
<?php echo xlt("There are required background services that are needed to send claims, pick up reports, and check eligibility. They are listed below in a table, but if there is something strange going on use the button to re-create the records."); ?>
```

Reply:
done